systemtest/src/test/java/io/strimzi/systemtest/cruisecontrol/CruiseControlApiST.java

```diff
@@ -30,6 +31,7 @@
 @Tag(REGRESSION)
 @Tag(CRUISE_CONTROL)
 @TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+@Disabled
```

Comment:
Here write some commend why do you Disabled these tests and same in ConfigurationST...Thanks

Reply:
I added to all 3 suites :) thanks for this reminder